homing_missiles.py

```diff
@@ -1,40 +1,64 @@
 import pygame
-from wordgenerator import WordGenerator
+
 from targeting_terminal import TargetingTerminal
+from wordgenerator import WordGenerator
 
 pygame.font.init()
 
+"""
+Colours
+"""
 BLACK = 0, 0, 0
-RED = 255, 0, 0
-GREEN = 0, 255, 0
 BLUE = 0, 0, 255
-YELLOW = 255, 255, 0
+GREEN = 0, 255, 0
+RED = 255, 0, 0
 WHITE = 255, 255, 255
+YELLOW = 255, 255, 0
+
+"""
+Event Keys
+"""
 EVENT_KEY_a = 97
-EVENT_KEY_z = 122
-EVENT_KEY_ENTER = 13
 EVENT_KEY_BACKSPACE = 8
+EVENT_KEY_ENTER = 13
+EVENT_KEY_z = 122
 
 
 class HomingMissilesTargetingSystem():
 
     def __init__(self, universe, DRAWING_SCALE):
-        self.word_generator_ = WordGenerator()
         self.universe_ = universe
         self.DRAWING_SCALE_ = DRAWING_SCALE
-        self.ui_font_ = pygame.font.SysFont("monospace", 15*DRAWING_SCALE)
+
+        self.current_text_ = ""
         self.enemy_color_ = YELLOW
+        self.font_size_ = 15
         self.main_character_color_ = GREEN
         self.target_tags_ = dict()
-        self.current_text_ = ""
+        self.target_tag_y_spacing_ = 5
         self.targeting_terminal_ = TargetingTerminal(DRAWING_SCALE)
+        self.text_antialias_ = 1
+        self.ui_font_ = pygame.font.SysFont("monospace", self.font_size_*DRAWING_SCALE)
+        self.word_generator_ = WordGenerator()
+        self.word_length_min_ = 12
+        self.word_length_range = 1
+
+    """
+    Update Functions
+    """
 
     def update(self, events):
         for enemy in self.universe_.enemies_.values():
             if enemy.ID_ not in self.target_tags_:
-                self.target_tags_[enemy.ID_] = self.new_word()
+                self.target_tags_[enemy.ID_] = \
```

Comment:
Lol wow thats a cool trick with "\". You are using it to escape the space right?